RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -95,6 +95,21 @@
     CAPhiCut = 0.05,
 ))
 
+trackingPhase2PU140.toModify(lowPtTripletStepHitDoublets, layerPairs = [0,1]) # layer pairs (0,1), (1,2)
+trackingPhase2PU140.toReplaceWith(lowPtTripletStepHitTriplets, _caHitTripletEDProducer.clone(
```

Comment:
I suggest to have a new round of cleanup of all config of Iterative tracking once PixelPair for Phase1 is also integrated: You may have noticed that I waited PU70 cleanup before even thinking to submit this PR.
At some point we also need to migrate Phase2 to MVA